src/core_modules/capture-core/components/D2Form/SectionDescriptionBox.types.ts

```diff
@@ -0,0 +1,7 @@
+import type { WithStyles } from '@material-ui/core/styles';
+
+export type Props = {
+    description: string;
+};
+
+export type PlainProps = Props & WithStyles<any>;
```

Comment:
Delete PlainProps and fix the error by importing and using `props: Props & WithStyles<typeof styles>` in src/core_modules/capture-core/components/D2Form/SectionDescriptionBox.component.tsx on line 27